sdk/monitor/monitor-opentelemetry-exporter/test/internal/statsbeat.spec.ts

```diff
@@ -43,19 +41,17 @@ describe("#AzureMonitorStatsbeatExporter", () => {
 
   describe("Export/Statsbeat", () => {
     let scope: nock.Interceptor;
-    let sandbox: any;
     const envelope = {
       name: "Name",
       time: new Date(),
     };
 
-    before(() => {
+    beforeAll(() => {
```

Comment:
was this migrated automatically or manually? If manually, I can add this to the list of codemods

Reply:
This was manual I believe